skyportal/tests/frontend/test_bokeh_plots_frontend.py

```diff
@@ -16,22 +16,23 @@ def expand_shadow_element(driver, element):
 def test_export_bold_light_curve_as_csv_button(driver, user, public_source):
     driver.get(f"/become_user/{user.id}")
     driver.get(f"/source/{public_source.id}")
-    driver.wait_for_xpath('//div[@class=" bk-root"]', timeout=20)
```

Comment:
Change name of this to test_plotly_plots_frontend?

Reply:
good idea. We might want to still test the export to CSV but via the photometry table